src/main/java/net/fabricmc/loom/util/Checksum.java

```diff
@@ -96,6 +96,7 @@ public static String toHex(byte[] bytes) {
 
 	public static String projectHash(Project project) {
 		String str = project.getProjectDir().getAbsolutePath() + ":" + project.getPath();
-		return toHex(str.getBytes(StandardCharsets.UTF_8)).substring(0, 16);
+		String hex = toHex(str.getBytes(StandardCharsets.UTF_8));
```

Comment:
I think the intended code whould be to hash the string, and then substring, eg. use `sha1Hex` instead of `hex`

Reply:
i was just using the original code. but i can make it use the hash version instead in this pr if thats fine

Reply:
If you could that would be great. I think it better fits what this code was intended to to, especially considering the name of the method.